resources/views/packs/index.blade.php

```diff
@@ -66,5 +66,6 @@
                 @endforeach
             </div>
         </div>
+        @include('objects._pagination', ['object' => $packs, 'modifier' => 'type'])
```

Comment:
Pagination should probably be inside the above container instead of between container and the footer (pagination is always within the container elsewhere on the site)